src/bindings/pygaia/scripts/classification/generate_params_report.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2006-2019  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Gaia
+#
+# Gaia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+import os.path, csv, argparse
+from gaia2.fastyaml import yaml
+from get_classification_results import ClassificationResults
+from optparse import OptionParser
+
+
+def generateParamsReport(project_file):
+    project = yaml.load(open(project_file, 'r'))
+    results_dir = project['resultsDirectory']
+    csv_file = project_file.rstrip('project') + 'report.tsv'
```

Comment:
here, is `project_file` a path like /path/to/directory/project?

It'd be better to get the directory of project_file with os.path.dirname, and then use os.path.join to add report.tsv to the path